cpp/src/arrow/csv/options.h

```diff
@@ -166,6 +166,9 @@ struct ARROW_EXPORT ReadOptions {
   /// Create read options with default values
   static ReadOptions Defaults();
 
+  /// weather to use an IO thread to read ahead
```

Comment:
```suggestion
  /// Whether to use an IO thread to read ahead while other blocks are decoding
```

Reply:
(feel free to reword differently if you find a better wording)

Reply:
Done